.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules
```

Comment:
- [ ] According to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), we should ignore `test.md` file by git. The .gitignore file is a text file that tells Git which files or folders to ignore in a project.

- [ ] [REQUIRED] - Please kindly include the `test.md` in the `.gitignore` file. 👍

- [ ] Although the `test.md` file is missing in your repo which is correct, maybe that is because you have already removed the file manually from your repo or you haven't pushed it there initially. 

- [ ] However, If you want to ignore a file/folder in your repository's root directory. we simply include it in the `.gitignore` file